### ISSUE TITLE
Make Travis automatically open a special issue in case of failing master

### ISSUE THREAD

**Issue by delapuente** [Qiskit/qiskit-terra#926]
### What is the expected enhancement?
The current merging process has a bottleneck explained on #927. One of the solutions involves targeting only one Python version for only one platform during regular merges and complete with a periodic build for testing the complete environment matrix.

For implementing this, we want some way to highlight when the _cron_ build is failing for some reason. We opted for instructing the CI to open an issue in the repository. The following format is provided as a reference and open for dicussión:

```
Title: Master is failing in <CI name> at <short commit hash>
Body: See details at: <link to logs in the CI>
Labels: "master failing"
```

**Comment by delapuente** [comment]
Well, this is what I propose:
1. Let `hash` be the commit hash causing the failure.
2. On failure, CI checks if there is an open issue with the label "master failing"
3. If it does not exists, open a new issue with the following format:
> Title: Master is failing
> Body: Master is failing in commit &lt;hash&gt;. Check the logs at: &lt;link to CI logs&gt;.
> Labels: "master failing"
4. Else, check if there is a comment containing `hash`.
5. If there is none, send a new comment with the following content:
> Master is failing in commit &lt;hash&gt;. Check the logs at: &lt;link to CI logs&gt;.

**Comment by mtreinish** [comment]
@delapuente I'm personally not necessarily convinced of the value of something like this. While I agree it could definitely highlight something real that needs our attention, like a new version of dependency package broke something or we introduced a bug causing non-deterministic failures, it is equally as likely that the travis infrastructure just fell apart. I've seen many times where the container they use for testing just fails to deploy, or a network issue fails an early setup stage and the job errors before even installing the project under test. Do we want an issue automatically opened if that kind of thing happens?

**Comment by 1ucian0** [comment]
I this this was closed via #1071.
